<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OnlineMusicWidget</class>
 <widget class="QWidget" name="OnlineMusicWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OnlineMusicWidget</string>
  </property>
  <widget class="QGroupBox" name="searchGroup">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>661</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;
border-radius:10px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
border:1px solid white;
border-radius:5px;
</string>
    </property>
    <property name="text">
     <string>搜索歌曲名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>71</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>71</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#searchButton
{
	background:transparent;
	color:rgb(200, 200, 200);
	border:1px solid white;
	border-radius:5px;
}
QPushButton#searchButton:hover 
{	
	background:transparent;
	color:rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>933</x>
     <y>0</y>
     <width>145</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,2">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QPushButton" name="optionsButton">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="OnlineMusicResource.qrc">
        <normaloff>:/images/prefix1/images/options.png</normaloff>:/images/prefix1/images/options.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="minButton">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>38</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>38</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#minButton
{
	border: none;
	background-color: transparent;
	color: white;
}
QPushButton#minButton:hover
{
	border:none;
	border-radius:5px;
	background-color: grey;
	color: blue;
}</string>
      </property>
      <property name="text">
       <string>—</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="minimumSize">
       <size>
        <width>44</width>
        <height>38</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>44</width>
        <height>38</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#closeButton
{
	border: none;
	background-color: transparent;
	color: rgb(255, 0, 0);
}
QPushButton#closeButton:hover
{
	border:none;
	border-radius:5px;
	background-color:rgb(255, 0, 0);
	color: white;
}
QPushButton#closeButton:pressed {
    background-color: rgb(200, 0, 0);  
    color: white;
}</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="lyric">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>120</y>
     <width>411</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>120</y>
     <width>16</width>
     <height>371</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>16</width>
      <height>352</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="playerGroup">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>520</y>
     <width>881</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>711</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="sliderLayout">
     <item>
      <widget class="QLabel" name="currentTimeLabel">
       <property name="styleSheet">
        <string notr="true">color: grey;</string>
       </property>
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="progressSlider">
       <property name="styleSheet">
        <string notr="true">QSlider::groove:horizontal {
    background-color: #CCCCCC;
    height: 8px;
    border-radius: 4px;
}

QSlider::sub-page:horizontal {
    background-color: rgb(71, 213, 213); 
    border-radius: 4px;  
}

QSlider::add-page:horizontal {
    background-color: #CCCCCC;  
    border-radius: 4px; 
}

QSlider::handle:horizontal {
    background-color: white; 
    border: none;
    width: 16px;
    margin: -4px 0;
    border-radius: 8px; 
}</string>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="totalTimeLabel">
       <property name="styleSheet">
        <string notr="true">color: grey;</string>
       </property>
       <property name="text">
        <string>01:00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="authorLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="playLayout">
     <item>
      <widget class="QPushButton" name="prevButton">
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="OnlineMusicResource.qrc">
         <normaloff>:/images/prefix1/images/left.png</normaloff>:/images/prefix1/images/left.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playButton">
       <property name="styleSheet">
        <string notr="true">QPushButton#playButton{
background-color: transparent;
border:none;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="OnlineMusicResource.qrc">
         <normaloff>:/images/prefix1/images/play.png</normaloff>
         <normalon>:/images/prefix1/images/pause.png</normalon>:/images/prefix1/images/play.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="OnlineMusicResource.qrc">
         <normaloff>:/images/prefix1/images/right.png</normaloff>:/images/prefix1/images/right.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="volumeLayout">
     <item>
      <widget class="QPushButton" name="volumeButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:transparent;
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="OnlineMusicResource.qrc">
         <normaloff>:/images/prefix1/images/colume.png</normaloff>
         <normalon>:/images/prefix1/images/columeOff.png</normalon>:/images/prefix1/images/colume.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="volumeSlider">
       <property name="styleSheet">
        <string notr="true">QSlider::groove:horizontal {
    background-color: rgb(132, 132, 132);
    height: 8px;
    border-radius: 4px;
}

QSlider::sub-page:horizontal {
	background-color: rgb(241, 161, 0);
    border-radius: 4px;  
}

QSlider::add-page:horizontal {
    border-radius: 4px; 
}

QSlider::handle:horizontal {
    background-color: white; 
    border: none;
    width: 16px;
    margin: -4px 0;
    border-radius: 8px; 
}</string>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>authorLabel</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>titleLabel</zorder>
  </widget>
  <widget class="QPushButton" name="localMusicListButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>580</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;本地歌曲&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="OnlineMusicResource.qrc">
     <normaloff>:/images/prefix1/images/localMusicList.png</normaloff>:/images/prefix1/images/localMusicList.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>211</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="musicList">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>271</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black</string>
   </property>
  </widget>
  <zorder>playerGroup</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>searchGroup</zorder>
  <zorder>lyric</zorder>
  <zorder>scrollArea</zorder>
  <zorder>localMusicListButton</zorder>
  <zorder>textEdit</zorder>
  <zorder>textEdit_2</zorder>
  <zorder>musicList</zorder>
 </widget>
 <resources>
  <include location="OnlineMusicResource.qrc"/>
 </resources>
 <connections/>
</ui>
